### ISSUE TITLE
inline clock component

### ISSUE THREAD

**Issue by tidou44** [weareoutman/clockpicker#31]
I tested your picker and find it very nice.
Is it planned to use the clockpicker inline in a form, and not in a bootstrap popover. I would like to use it directly in a bootstrap modal.

Thank you in advance !

**Comment by thefont** [comment]
+1 I would really like to see this as well!

**Comment by archerchiu** [comment]
+1 this is a good idea.

**Comment by EHLOVader** [comment]
+1

**Comment by ghKZ** [comment]
+1

**Comment by rkuzovlev** [comment]
+1

**Comment by koendecock** [comment]
+1

**Comment by rogchap** [comment]
+1

**Comment by shubham3233** [comment]
+1

**Comment by dafik** [comment]
to get inline do this:
in function ClockPicker.prototype.show change :

`$body = $(document.body).append(this.popover);` 
to 
`this.popover.insertAfter(this.element);`

and  in ClockPicker.prototype.locate:
`offset = element.offset(),`
to
`offset = {top:0,left:0},`

**Comment by traykovn** [comment]
+1

**Comment by traykovn** [comment]
Good idea, but currently it is pretty tight to a popover. It won't be easy to implement I guess.

**Comment by papayaah** [comment]
thanks @dafik your solution actually works. was able to render them inline with your code:

![image](https://user-images.githubusercontent.com/175045/27771656-d2003804-5f85-11e7-8c12-1686a060c4f1.png)

no need for the offset though.

**Comment by uitlaber** [comment]
Need inline component +1
